packages/cli/test/mocks/project.ts

```diff
@@ -131,6 +131,9 @@ export function useProject(project: Partial<Project> = defaultProject) {
   client.scenario.get(`/v8/projects/${project.id}`, (_req, res) => {
     res.json(project);
   });
+  client.scenario.get(`/v8/projects/:nameOrId`, (_req, res) => {
```

Comment:
Why is this one necessary? This will make `project` be returned even with a non-matching ID. Seems like maybe the test isn't set up correctly if this is needed.